packages/components/psammead-brand/src/index.test.jsx

```diff
@@ -78,6 +78,23 @@ describe('Brand', () => {
     />,
   );
 
+  shouldMatchSnapshot(
+    'should render correctly with a link id',
+    <Brand
+      product="BBC News"
+      svg={svg}
+      url="https://www.bbc.co.uk/news"
+      svgHeight={24}
+      maxWidth={280}
+      minWidth={180}
+      borderTop
+      borderBottom
+      backgroundColour={C_POSTBOX}
+      logoColour={C_WHITE}
+      linkId="brandLink"
```

Comment:
I'd consider using an explicit test rather than snapshot here, perhaps mimicking the code you will use in Simorgh to focus on the element.

Reply:
Yeah good idea 👍 In order to fully mimic the Simorgh code, I'll have to use the amp `<scripts>` to get amp focus action working, is this ok? Or do you think it'd be acceptable to just call `.focus()` using the DOM API?

Reply:
Just using the regular DOM api seems okay to me. Tbh this comment is more about my dislike of snapshot tests, even a test that just used a query selector to find an element with a the `brandLink` id would be enough for me. The main point is that no one can just update the snapshot, the test will fail if the id isn't on the link for some reason.

Reply:
Ah, I see, makes sense 👍 Thanks